search-service/src/main/kotlin/com/egm/stellio/search/util/QueryUtils.kt

```diff
@@ -0,0 +1,52 @@
+package com.egm.stellio.search.util
+
+import com.egm.stellio.search.model.TemporalQuery
+import com.egm.stellio.search.service.AttributeInstanceService
+import com.egm.stellio.search.service.TemporalEntityAttributeInstancesResult
+import com.egm.stellio.search.service.TemporalEntityAttributeService
+import com.egm.stellio.search.service.TemporalEntityService
+import com.egm.stellio.shared.model.CompactedJsonLdEntity
+import com.egm.stellio.shared.util.*
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import java.net.URI
+import java.util.*
+
+@Service
+class QueryUtils(
```

Comment:
It looks more like a service than what we usually put in utils folder (typically stateless functions). It even has a `@Service` annotation.